inject/src/main/java/io/micronaut/context/ApplicationContext.java

```diff
@@ -59,11 +58,6 @@
  */
 public interface ApplicationContext extends BeanContext, PropertyResolver, PropertyPlaceholderResolver {
 
-    /**
-     * @return The default conversion service
-     */
-    @NonNull ConversionService<?> getConversionService();
```

Comment:
Why was this deleted?

Reply:
It's now defined in `ConversionServiceProvider`, which the context extends.